editions/tw5.com/tiddlers/widgets/ActionListopsWidget.tid

```diff
@@ -49,12 +35,12 @@ For example, the items "abc" and "123" can be appended to the field `myfield` us
 The same can be achieved using the `$filter` attribute and prepending the [[Filter Run]] `[all[current]get[myfield]enlist-input[]]` to the [[Filter Expression]]:
```

Comment:
Sorry for the late comment, but this line should also have been changed to use `[enlist{!!myfield}]` so it matches the change you made in the next line.

Reply:
Another PR would be welcome, of course

Reply:
🙃